### ISSUE TITLE
Please release the Raspberry Pi Demo

### ISSUE THREAD

**Issue by ArriFerrari** [tensorflow/tensorflow#25903]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: 1.12
- Doc Link: https://youtu.be/FAMfy7izB6A?t=930

**Describe the documentation issue**
Andrew Selle showed off a cool demo, and said, "I'm not an electrical engineer or mechanical engineer, so you can do this too." It's been 10.5 months. Maybe he forgot to release the code so that we can do that too? He also mentioned in the comment section that it would happen.

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
I don't have access to the code in question

**Comment by Ayush517** [comment]
@dynamicwebpaige @aselle Can I please work on this as a project for this year's GSOC.

**Comment by leigh-johnson** [comment]
Hi y'all! @Ayush517 @dynamicwebpaige @aselle 

@Ayush517, did you end up working on this project for Google's Summer of Code?

**Comment by Ayush517** [comment]
@leigh-johnson No,  I am working on "Tutorials and Guides for Swift for TensorFlow" for Google Summer of Code.
A similar thing was I think mentioned in the mailing list. You can find it [here](https://groups.google.com/a/tensorflow.org/forum/?nomobile=true#!topic/tflite/Lx6VrFmX_4s)

**Comment by nikitamaia** [comment]
Hi @ArriFerrari, are you still interested in getting access to this demo?

**Comment by ArriFerrari** [comment]
Hi Nikita,
Yes! It would still be helpful!
-ArriFerrari

**Comment by mohantym** [comment]
Hi @ArriFerrari ! I have attached few relevant examples on camera based tracking in Rasbperry pi  for reference. Link [1](https://medium.com/augmented-startups/raspberry-pi-pan-tilt-tracking-camera-using-opencv-ai-kit-tutorial-app-4-49bf122401a4), [2](https://www.pyimagesearch.com/2019/04/01/pan-tilt-face-tracking-with-a-raspberry-pi-and-opencv/), [3](https://www.instructables.com/Automatic-Vision-Object-Tracking/) ,[4](https://opensource.com/article/20/1/object-tracking-camera-raspberry-pi) Hope it helps address this issue.Thank you!

**Comment by ArriFerrari** [comment]
Thank you so much! I’ve been wanting to try this for a while.

**Comment by aselle** [comment]
At this point I think the code has not been updated, and so it is unfortunately unlikely it will be released. I'm happy to answer any questions if people are trying to do something similar.
